.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Auto release 🍝
```

Comment:
I am not convinced that this step is required as in the current setup the auto release is invoked automatically when the tag is pushed, [see](https://github.com/gizmokis/resin/blob/7bd149616f5dd9b178c9beffca3bf54266b23573/.github/workflows/linux.yml#L68). If you wish you can replace the old strategy with the new one proposed here.